docs/hacking/install.rst

```diff
@@ -186,64 +171,139 @@ system that Docker can be installed on:
 
    .. code-block:: bash
 
-      docker logs nsqd
+      sudo docker logs nsqd
 
    For more on how to use Docker see the `Docker website`_.
 
 
 .. _Docker website: https://www.docker.com/
 
 
-Running h
----------
+Installing the gulp command
```

Comment:
Gulp is used in multiple places further down the page, so moving it here removed duplication and also I think it's clearer to have separate steps like "install gulp" in separate sections. https://github.com/hypothesis/h/pull/3219/commits/c55ba7ba0ba219490b8f75c58afc262171f4b8d4